composer/utils/object_store/sftp_object_store.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utility for uploading to and downloading from cloud object stores."""
+import os
+import uuid
+from typing import Optional
+
+from composer.utils.import_helpers import MissingConditionalImportError
+from composer.utils.object_store.object_store import ObjectStore, ObjectStoreTransientError
+
+__all__ = ['SFTPObjectStore']
+
+
+class SFTPObjectStore(ObjectStore):
+    """Utility for uploading to and downloading to a server via SFTP.
+
+    Args:
+        host (str): The server to connect to
+        port (int): The server port to connect to
+        username (str): The username to authenticate
+        key_file_path (str): The filename of private key
+        cwd (Optional[str]): The directory to navigate to upon creating the SSH connection.
+    """
+
+    def __init__(self, host: str, port: int, username: str, key_file_path: str, cwd: Optional[str] = None):
```

Comment:
Can you add in a `kwargs` argument so the user can pass through additional options into paramiko?

Reply:
Looking though the Paramiko functions I use - none have extra args to pass.